source/components/molecules/CheckboxField/CheckboxField.stories.js

```diff
@@ -29,49 +29,50 @@ const CheckboxFields = () => {
   return (
     <View>
       <CheckboxField
-        text="Do you feel it? Light, small"
-        color="light"
+        text="Do you feel it? Blue theme, small"
+        color="blue"
         size="small"
-        checked={checkboxValues.box1}
+        value={checkboxValues.box1}
         onChange={handleChange('box1')}
+        help={{ text: 'some other helper text' }}
       />
       <CheckboxField
         text="Does it feel good? Can I ask longer questions here? How does it handle long strings?"
-        color="gray"
+        color="red"
         size="small"
-        checked={checkboxValues.box2}
+        value={checkboxValues.box2}
         onChange={handleChange('box2')}
+        help={{ text: 'some helper text' }}
       />
       <CheckboxField
-        text="Is it happening? Light, medium"
-        color="light"
+        text="Is it happening? Green theme, medium"
+        color="green"
         size="medium"
-        checked={checkboxValues.box5}
+        value={checkboxValues.box5}
         onChange={handleChange('box5')}
       />
-      <View backgroundColor="rgb(29,32,37)">
-        <CheckboxField
-          text="Do you feel it? Dark theme"
-          color="dark"
-          size="small"
-          checked={checkboxValues.box3}
-          onChange={handleChange('box3')}
-        />
-        <CheckboxField
-          text="Is it happening? Blue theme"
-          color="blue"
-          size="large"
-          checked={checkboxValues.box4}
-          onChange={handleChange('box4')}
-        />
-        <CheckboxField
-          text="Is it happening? Blue theme"
-          color="blue"
-          size="large"
-          checked={checkboxValues.box6}
-          onChange={handleChange('box6')}
-        />
-      </View>
+      <CheckboxField
+        text="Do you feel it? Purple theme, small"
+        color="purple"
```

Comment:
Invalid prop type: Expected one of the colors defined in theme -> checkboxField.

Reply:
Fixed the proptype of colors to use the new theme colors.